mandatory/1-fix-functions.js

```diff
@@ -11,29 +11,26 @@
 */
 
 function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+  let isHappy;
+  if (isHappy = true) {
```

Comment:
This if statement is not evaluating that ```isHappy``` is true. It attempting to assign the value ```true``` to ```isHappy```. This function should also be taking in an argument.